components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -28,8 +28,14 @@ import java.util.concurrent.TimeUnit
 import java.util.zip.GZIPOutputStream
 import kotlin.random.Random
 
-private val defaultServerUrl = "https://crash-reports.mozilla.com/submit?" +
+private const val DEFAULT_SERVER_URL = "https://crash-reports.mozilla.com/submit?" +
         "id=${BuildConfig.MOZ_APP_ID}&version=${BuildConfig.MOZILLA_VERSION}&${BuildConfig.MOZ_APP_BUILDID}"
+
+@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+internal const val CAUGHT_EXCEPTION_PREFIX = "Not a crash"
+
+@VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+internal const val UNCAUGHT_EXCEPTION_PREFIX = "Fatal crash"
```

Comment:
Are those for distinguishing both types on Crashstats? I wonder if we should use some kind of prefix like on mailing lists, e.g. `[CRASH]`. Not sure about the other though. :) `[INFO]`, `[REPORT]`, ..

Reply:
I'll use the [INFO] tag.  Assuming we won't put a prefix for the uncaught exception since we don't want to affect uncaught exceptions.